README.rst

```diff
@@ -208,6 +208,28 @@ Examples:
 
 ``fades -d requests --pip-options="--index-url="http://example.com"``
 
+Setting options using config files
+----------------------------------
+
+You can also configure fades using `.ini` config files. fades will search for config files in 
+`/etc/fades.ini`, `~/.fades` and `.fades`. 
```

Comment:
Are you sure? I was thinking in config files like .vimrc .bashrc .gitconfig .tmux.conf, etc.

Also I should use `os.path.expanduser('~')`